hieradata/production.yaml

```diff
@@ -561,9 +561,6 @@ hosts::production::router::hosts:
     ip: '10.3.1.254'
     legacy_aliases:
       - 'cache'
-      - "www.%{hiera('app_domain')}"
-      - "www-origin.%{hiera('app_domain')}"
-      - "assets-origin.%{hiera('app_domain')}"
```

Comment:
How is this hieradata relevant, as I didn't think the router or cache machines existed any longer in Carrenza?

Reply:
Yeah, I don't think they do exist, but as I understand it this Hieradata creates entries in `/etc/hosts` meaning that we can't access services using the external domain.

```
thomasleese@production-backend-1:~$ cat /etc/hosts | grep www-origin
10.3.1.254	cache.router.publishing.service.gov.uk	cache.router cache.cluster router.cluster cache www.publishing.service.gov.uk www-origin.publishing.service.gov.uk assets-origin.publishing.service.gov.uk

thomasleese@production-backend-1:~$ curl https://www-origin.publishing.service.gov.uk
curl: (7) Failed to connect to www-origin.publishing.service.gov.uk port 443: Connection timed out
```

Reply:
I suppose we can actually remove the whole block about the cache machines, but I didn't want to remove too much as I'm not familiar enough with this part of the infrastructure configuration.

Reply:
Ok, good to know.

I ended up in this minefield yesterday... https://github.com/alphagov/govuk-puppet/pull/9061